packages/block-editor/src/components/spacing-panel-control/index.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { PanelBody } from '@wordpress/components';
+import { useSelect } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import InspectorControls from '../inspector-controls';
+
+export default function SpacingPanelControl( { children, ...props } ) {
+	const isSpacingEnabled = useSelect( ( select ) => {
+		const { getSettings } = select( 'core/block-editor' );
+		return get( getSettings(), '__experimentalEnableCustomSpacing' );
```

Comment:
I think you could use optional chaining here instead of Lodash `get`:
```suggestion
		return getSettings()?.__experimentalEnableCustomSpacing
```

Reply:
I copied this implementation from other places in the codebase with `get( getSettings(), ... ) `. I think it would be better to keep it consistent for now. We can refactor later if required ✌️